src/Stripe.net/Entities/Invoices/Invoice.cs

```diff
@@ -52,11 +52,10 @@ public class Invoice : StripeEntity<Invoice>, IHasId, IHasMetadata, IHasObject
         public bool AutoAdvance { get; set; }
 
         /// <summary>
-        /// One of <see cref="Billing" />. When charging automatically, Stripe will attempt to pay
-        /// this subscription at the end of the cycle using the default source attached to the
-        /// customer. When sending an invoice, Stripe will email your customer an invoice with
-        /// payment instructions.
+        /// This field has been renamed to collection_method and will be removed
```

Comment:
So it should say `CollectionMethod` since you're in .NET and it should be in <c></c>

Reply:
:)